### DISCUSSION TITLE
Github should be banned these account

### DISCUSSION THREAD

**Discussion by fdciabdul** [community/community#35498] [Other Feature Feedback, Questions, & Ideas] [answered]
please banned auto following bot account

**Comment by fdciabdul** [comment]
![Screenshot_20221008-114158576](https://user-images.githubusercontent.com/31664438/194688831-a9d40161-d9a0-4d1c-bdb7-f475d85bd984.jpg)
![Screenshot_20221008-114146053](https://user-images.githubusercontent.com/31664438/194688838-a953fdca-9fb5-42d2-8a2e-d1650bababf8.jpg)

**Comment by airtower-luna** [comment]
See the "Block or Report" lines in the screenshot? You can click there and fill the form to report the accounts to support. Us other users can't ban anyone. :wink:

**Reply by fdciabdul** [reply]
ik , but the bot isn't the only one or two account , but too much , hope github will automatic detect them

**Comment by queenofcorgis** [comment, marked as answer]
Hi @fdciabdul  👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
